ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/TransactionCommitterSpec.scala

```diff
@@ -196,6 +196,35 @@ class TransactionCommitterSpec extends WordSpec with Matchers with MockitoSugar
     }
   }
 
+  "buildLogEntry" should {
+    "produce an out-of-time-bounds rejection log entry in case pre-execution is enabled" in {
+      val context = new FakeCommitContext(recordTime = None)
+
+      val actualSuccessLogEntry = instance.buildLogEntry(aTransactionEntrySummary, context)
```

Comment:
Why are the success log entry assertions present only in this test and not in others?

Reply:
Added test cases for checking just the success log entry independent from pre-execution (it's a separate concern).